src/main/java/com/entropy/entropay/auth/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.entropy.entropay.auth;
+
+public class Constants {
```

Comment:
this can be an interface. But do we really need to start with a class that can end up being a bag of values not related to them? I think that for now, we can have them defined as constants in the relevant class